Python/executor_cases.c.h

```diff
@@ -118,12 +118,38 @@
             break;
         }
 
+        case TO_BOOL: {
+            static_assert(INLINE_CACHE_ENTRIES_TO_BOOL == 3, "incorrect cache size");
+            PyObject *value = stack_pointer[-1];
+            PyObject *res;
+            #line 296 "Python/bytecodes.c"
+            #if ENABLE_SPECIALIZATION
+            _PyToBoolCache *cache = (_PyToBoolCache *)next_instr;
+            if (ADAPTIVE_COUNTER_IS_ZERO(cache->counter)) {
+                next_instr--;
+                _Py_Specialize_ToBool(value, next_instr);
+                DISPATCH_SAME_OPARG();
+            }
+            STAT_INC(TO_BOOL, deferred);
+            DECREMENT_ADAPTIVE_COUNTER(cache->counter);
+            #endif  /* ENABLE_SPECIALIZATION */
+            int err = PyObject_IsTrue(value);
+            #line 138 "Python/executor_cases.c.h"
+            Py_DECREF(value);
+            #line 308 "Python/bytecodes.c"
+            if (err < 0) goto pop_1_error;
+            res = err ? Py_True : Py_False;
+            #line 143 "Python/executor_cases.c.h"
+            stack_pointer[-1] = res;
+            break;
+        }
+
         case TO_BOOL_BOOL: {
             PyObject *value = stack_pointer[-1];
             #line 313 "Python/bytecodes.c"
             DEOPT_IF(!PyBool_Check(value), TO_BOOL);
             STAT_INC(TO_BOOL, hit);
-            #line 127 "Python/executor_cases.c.h"
+            #line 153 "Python/executor_cases.c.h"
```

Comment:
Useless diff chunks like this make it harder to review the generated output. I would love to drop the `#line` directives. (Unsurprisingly, I get a lot of value out of seeing the diff of the generated code when I am fiddling with the code generator.) For debugging you could still generate line numbers using
```
python3 Tools/cases_generator/generate_cases.py -lPy_BUILD_CORE
```